src/js/components/Pagination/Pagination.js

```diff
@@ -194,24 +208,34 @@ const Pagination = forwardRef(
     return (
       <>
         {children && currentItems.map(item => children(item))}
-        <Nav
-          a11yTitle={a11yTitle || 'Pagination Navigation'}
-          ref={ref}
-          {...rest}
-        >
-          <Box
-            as="ul"
-            align="center"
-            direction="row"
-            gap="xxsmall"
-            pad="none"
-            margin="none"
+        {/* Do we want pontential for internally displaying a "results summary"?
+         If so, applying the container props to this box now may leave better
+          flexibility moving forward
+        */}
+        <StyledPaginationContainer {...theme.pagination.container}>
+          {/* Ideation: internal control of some results summary, need to think 
+          about what the default would be and how it may be customized
+          */}
+          {/* {showSummary && <Text>Showing 1-10 of 500 results</Text>} */}
+          <Nav
+            a11yTitle={a11yTitle || 'Pagination Navigation'}
+            ref={ref}
+            {...rest}
           >
-            {pages.map(page => (
-              <PageIndex key={page.page} {...page} />
-            ))}
-          </Box>
-        </Nav>
+            <Box
+              as="ul"
+              align="center"
+              direction="row"
+              gap="xxsmall"
+              pad="none"
+              margin="none"
```

Comment:
Should we account for these in the theme as defaults, rather than hardcoding here?

Reply:
Do pad and margin even need to be specified here?

Reply:
pad and margin need to be specified because "ul" comes with some default values. However, I do think we can make this a styled component and move the defaults to theme.

Reply:
👍